crates/sui-core/src/authority_batch.rs

```diff
@@ -210,7 +210,7 @@ impl crate::authority::AuthorityState {
                     .tables
                     .batches
                     .insert(&new_batch.data().next_sequence_number, &new_batch)?;
-                debug!(next_sequence_number=?new_batch.data().next_sequence_number, "New batch created");
+                debug!(next_sequence_number=?new_batch.data().next_sequence_number, "New batch created. Transactions: {:?}", current_batch);
```

Comment:
I think we should move this to trace.  As the system scales up, this will produce a huge log with tons of TX digests....  maybe we can dump just the first few and do `....` for the rest (with the number of digests)